### PR TITLE
Comet Boots use electric magic for fall damage reduction

### PR DESCRIPTION
**Pull request by Alastors**
Uses a similar (but altered) formula to how EMT calculates its boots fall damage.

### CODE REVIEW

**Comment by boubou19** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14402

File: src/main/java/thaumicboots/item/boots/comet/ItemElectricCometBoots.java [hunk 1]
```diff
@@ -122,8 +122,18 @@ public void onArmorTick(World world, EntityPlayer player, ItemStack itemStack) {
             } else {
                 player.jumpMovementFactor = 0.05F;
             }
-            if (player.fallDistance > 0.25F) {
-                player.fallDistance -= 0.25F;
+            if (player.fallDistance > 4.0F) {
+                player.fallDistance -= 0.50F;
+                if ((player.fallDistance > 3.5F)) {
```
Comment (eigenraven) [inline src/main/java/thaumicboots/item/boots/comet/ItemElectricCometBoots.java:127]: these double conditionals make no sense, they're always true if the outer one is true:
if x>4, then x-0.5 > 3.5, same in the other files

Comment (Alastors) [reply src/main/java/thaumicboots/item/boots/comet/ItemElectricCometBoots.java:127]: It’s checking the fall damage height no? It’s checking if it’s greater than 4 (which is confirmed to work) then if it is, reducing it by a half block, and if it is still above 3.5 blocks it takes electric damage up to the limit

Comment (Alastors) [reply src/main/java/thaumicboots/item/boots/comet/ItemElectricCometBoots.java:127]: At least that’s the intention

Comment (boubou19) [reply src/main/java/thaumicboots/item/boots/comet/ItemElectricCometBoots.java]: Raven is right, if the first inequality is verified, then the second is always verified

**Comment by boubou19** [comment]
superseded by #4

File: src/main/java/thaumicboots/item/boots/comet/ItemElectricCometBoots.java (hunk 1 above)
Comment (Alastors) [reply src/main/java/thaumicboots/item/boots/comet/ItemElectricCometBoots.java]: Yeahhh didn't see that until they mentioned it, thanks for fixing it

**Pull request closed without merging**